vue-app/src/views/Landing.vue

```diff
@@ -204,6 +210,16 @@ export default class Landing extends Vue {
       .getElementById('section-how-it-works')
       ?.scrollIntoView({ behavior: 'smooth' })
   }
+
+  get chain(): any {
```

Comment:
Same deal here - if we need this in multiple places, should we just create util functions in networks.ts?

Reply:
> Also, side note related to `networks.ts`... noticing it really only checks for XDAI and we now have a setup in `chains.ts` that can return the chains native currency. I think we should patch this so it looks up from `chains.ts` and returns `.currency`... trying to draw the line on this PR though.

Yes, that makes sense.